### ISSUE TITLE
Inquiry for the release date of the pre-trained model

### ISSUE THREAD

**Issue by siyuhsu** [princeton-nlp/AutoCompressors#4]
Hi, the pre-trained model mentioned in the repository is currently not available. Is there a tentative release date for the pre-trained model? It would be great to have an estimated date.

**Comment by CodeCreator** [comment]
Hi! Apologies for the delay. The models from the paper can now be found on huggingface hub and are linked on the README:

|Link|Base model|Fine-tuning seq. length|Fine-tuning data|#Summary vectors|Summmary accumulation|Randomized segmenting|Softprompt stop gradient|
|-|-|-|-|-|-|-|-|
| [princeton-nlp/AutoCompressor-2.7b-6k](https://huggingface.co/princeton-nlp/AutoCompressor-2.7b-6k) | OPT-2.7b | 6144 tokens / seq. in 4 compression steps | 2B tokens across 4 domains from the Pile | 50 / compression step | ✔️ | ✔️ | ✔️ |
| [princeton-nlp/RMT-2.7b-8k](https://huggingface.co/princeton-nlp/RMT-2.7b-8k) | OPT-2.7b | 8192 tokens / seq. in 4 compression steps | 2B tokens across 4 domains from the Pile | 50 | | | |
| [princeton-nlp/FullAttention-2.7b-4k](https://huggingface.co/princeton-nlp/FullAttention-2.7b-4k) | OPT-2.7b | 4092 tokens / seq. without compression | 2B tokens from 4 domains from the Pile | - | | | |
| [princeton-nlp/AutoCompressor-2.7b-30k](https://huggingface.co/princeton-nlp/AutoCompressor-2.7b-30k) | OPT-2.7b | 30720 tokens / seq. in 20 compression steps | 2B tokens from Books3 from the Pile | 50 / compression step | ✔️ | ✔️ | ✔️ |
| [princeton-nlp/AutoCompressor-1.3b-30k](https://huggingface.co/princeton-nlp/AutoCompressor-1.3b-30k) | OPT-1.3b | 30720 tokens / seq. in 20 compression steps | 2B tokens from Books3 from the Pile | 50 / compression step | ✔️ | ✔️ | ✔️ |
| [princeton-nlp/AutoCompressor-1.3b-30k](https://huggingface.co/princeton-nlp/RMT-1.3b-30k) | OPT-1.3b | 30720 tokens / seq. in 15 compression steps | 2B tokens from Books3 from the Pile | 50 | | | |
